### PR TITLE
docs: update `openid-connect` plugin `refresh_session_interval` attribute

### PR DESCRIPTION
**Pull request by baqianxin**
### Description

There are some attributes available in the [lua-resty-openidc](https://github.com/zmartzone/lua-resty-openidc) that are not mentioned in the plugin schema and docs. They are configurable, however, and passed down to [lua-resty-openidc](https://github.com/zmartzone/lua-resty-openidc) through opts.

Quick reference:
https://github.com/apache/apisix/blob/master/apisix/plugins/openid-connect.lua
https://github.com/zmartzone/lua-resty-openidc
https://github.com/apache/apisix/pull/10591

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/plugins/openid-connect.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ description: OpenID Connect（OIDC）是基于 OAuth 2.0 的身份认证协议
 | set_refresh_token_header             | boolean | 否     | false                 |               | 当设置为 `true` 并且刷新令牌可用时，则会将该属性设置在`X-Refresh-Token`请求头中。                      |
 | session                              | object  | 否     |                       |               | 当设置 bearer_only 为 false 时，openid-connect 插件将使用 Authorization Code 在 IDP 上进行认证，因此你必须设置 session 相关设置。 |
 | session.secret                       | string  | 是     | 自动生成               | 16 个以上字符  | 用于 session 加密和 HMAC 计算的密钥。 |
+| refresh_session_interval                       | integer   | 否     | 900  |  | 900 秒后刷新用户存储的session，而无需进行重新授权流程                                                                                                                                                                         |
```
Comment (SylviaBABY) [inline docs/zh/latest/plugins/openid-connect.md:63]:
```suggestion
| refresh_session_interval                       | integer   | 否     | 900  |  | 900 秒后刷新用户存储的 session，而无需进行重新授权流程。                                                                                                                                                                         |
```

File: docs/en/latest/plugins/openid-connect.md [hunk 2]
```diff
@@ -61,6 +61,7 @@ description: OpenID Connect allows the client to obtain user information from th
...
 | session                              | object   | False    |                       |              | When bearer_only is set to false, openid-connect will use Authorization Code flow to authenticate on the IDP, so you need to set the session-related configuration.                                                                   |
 | session.secret                       | string   | True     | Automatic generation  | 16 or more characters | The key used for session encrypt and HMAC operation.                                                                                                                                                                                  |
+| refresh_session_interval                       | integer   | False     | 900  |  | Refresh the users id_token(or session in the browser) after 900 seconds without requiring re-authentication                                                                                                                                                                          |
```
Comment (SylviaBABY) [inline docs/en/latest/plugins/openid-connect.md:64]:
```suggestion
| refresh_session_interval                       | integer   | False     | 900  |  | Refresh the user's id_token(or session in the browser) after 900 seconds without requiring re-authentication.                                                                                                                                                                          |
```

**Comment by kayx23** [comment]
This is already added in https://github.com/apache/apisix/pull/10591/files

**Comment by baqianxin** [comment]
@kayx23 the ci on centos7 was faild,because the `TEST 24` case 

```bash
#   Failed test 't/plugin/openid-connect.t TEST 24: Check defaults. - response_body - response is expected (repeated req 0, req 0)'
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 1674.
#          got: '{"accept_none_alg":false,"accept_unsupported_alg":true,"access_token_expires_leeway":0,"access_token_in_authorization_header":false,"bearer_only":false,"client_id":"kbyuFDidLLm280LIwVFiazOqjO3ty8KH","client_jwt_assertion_expires_in":60,"client_secret":"60Op4HFM0I8ajz0WdiStAbziZ-VFQttXuxixHHs2R7r7-CW8GR79l-mmLqMhc-Sa","discovery":"http://127.0.0.1:1980/.well-known/openid-configuration","force_reauthorize":false,"iat_slack":120,"introspection_endpoint_auth_method":"client_secret_basic","introspection_interval":0,"jwk_expires_in":86400,"jwt_verification_cache_ignore":false,"logout_path":"/logout","realm":"apisix","renew_access_token_on_expiry":true,"revoke_tokens_on_logout":false,"scope":"openid","set_access_token_header":true,"set_id_token_header":true,"set_refresh_token_header":false,"set_userinfo_header":true,"ssl_verify":false,"timeout":3,"token_endpoint_auth_method":"client_secret_basic","unauth_action":"auth","use_nonce":false,"use_pkce":false}
# '
#     expected: '{"accept_none_alg":false,"accept_unsupported_alg":true,"access_token_expires_leeway":0,"access_token_in_authorization_header":false,"bearer_only":false,"client_id":"kbyuFDidLLm280LIwVFiazOqjO3ty8KH","client_jwt_assertion_expires_in":60,"client_secret":"60Op4HFM0I8ajz0WdiStAbziZ-VFQttXuxixHHs2R7r7-CW8GR79l-mmLqMhc-Sa","discovery":"http://127.0.0.1:1980/.well-known/openid-configuration","force_reauthorize":false,"iat_slack":120,"introspection_endpoint_auth_method":"client_secret_basic","introspection_interval":0,"jwk_expires_in":86400,"jwt_verification_cache_ignore":false,"logout_path":"/logout","realm":"apisix","refresh_session_interval":900,"renew_access_token_on_expiry":true,"revoke_tokens_on_logout":false,"scope":"openid","set_access_token_header":true,"set_id_token_header":true,"set_refresh_token_header":false,"set_userinfo_header":true,"ssl_verify":false,"timeout":3,"token_endpoint_auth_method":"client_secret_basic","unauth_action":"auth","use_nonce":false,"use_pkce":false}
# '
# Looks like you failed 1 test of 99.
[12:14:20] t/plugin/openid-connect.t .................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/99 subtests 
```

**Review by kayx23 — APPROVED** [review]

**Pull request merged**